src/styles.css

```
@@ -566,26 +566,47 @@ footer a:hover{text-decoration:underline}
   border: none;
-  font-size: 18px;
+  font-size: 20px;
   cursor: pointer;
-  display: none;
   z-index: 50;
-  box-shadow: 0 3px 10px rgba(244,107,14,.35);
-  transition: opacity .2s;
-}
-
-.scroll-top-btn.visible {
+  box-shadow: 0 4px 14px rgba(244,107,14,.35);
```

Stylelint (17.12.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)